code/modules/surgery/operations/operation_bioware.dm

```diff
@@ -247,9 +247,25 @@
 	name = "cortex folding"
 	rnd_name = "Encephalofractoplasty (Cortex Folding)" // it's a stretch - "brain fractal reshaping"
 	desc = "A biological upgrade which folds a patient's cerebral cortex into a fractal pattern, increasing neural density and flexibility."
-	status_effect_gained = /datum/status_effect/bioware/cortex/folded
+	operation_flags = OPERATION_AFFECTS_MOOD | OPERATION_NOTABLE | OPERATION_MORBID | OPERATION_LOCKED | OPERATION_NO_PATIENT_REQUIRED
+	status_effect_gained = /datum/status_effect/bioware/cortex // Not actually applied, simply for compatibility checks
 	required_zone = BODY_ZONE_HEAD
 
+/datum/surgery_operation/limb/bioware/cortex_folding/state_check(obj/item/bodypart/limb)
+	if(limb.body_zone != required_zone)
+		return FALSE
+	var/obj/item/organ/brain/brain = locate() in limb
+	if(isnull(brain))
+		return FALSE
+	return !HAS_TRAIT_FROM(brain, TRAIT_SPECIAL_TRAUMA_BOOST, BIOWARE_TRAIT)
```

Comment:
I'm pretty sure this should call parent, to check if we have status effect `/datum/status_effect/bioware/cortex` 

Otherwise we're not actually doing the compatibility check 

(+ removes the copypasta)